packages/former/src/components/formFieldOptions.ts

```diff
@@ -1,28 +1,56 @@
 import type { FormKitSchemaNode } from '@formkit/core';
 
+export const availableFieldTypes = ['text', 'number'];
```

Comment:
This is not related with the `formFieldOptions`. Might be better to have a separate list with component names (sth we can later translate) and schemas. Would also help to extend it with icons later on.

Reply:
@anbraten I do not fully understand your suggestion. Do you mean that the value that we assign to the schema when adding a new component (in the addComponent function) is predefined in a file with this availableFieldTypes list and we retrieve this schema based from that composition on the basis of selected type?

Reply:
Thanks. I've adjusted the list of available field types a bit again.